src/main/resources/wa-task-configuration-ia-asylum.dmn

```diff
@@ -84,6 +107,78 @@ else if caseData.appealType = "protection" then "Protection"
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1gm06lg">
+          <text>"reviewAppeal","followUpOverdueRespondentEvidence","reviewRespondentEvidence","followUpOverdueCaseBuilding","reviewAppealSkeletonArgument","followUpOverdueReasonsForAppeal","reviewReasonsForAppeal",       "followUpOverdueClarifyingAnswers","reviewClarifyingAnswers","followUpOverdueRespondentReview","reviewRespondentResponse","followUpOverdueCMARequirements","reviewCmaRequirements","reviewAdditionalHomeOfficeEvidence",           "reviewAdditionalAppellantEvidence","reviewAddendumHomeOfficeEvidence","reviewAddendumAppellantEvidence","reviewAddendumEvidence","processReviewDecisionApplication"</text>
+        </inputEntry>
+        <outputEntry id="LiteralExpression_0omxasv">
+          <text>"workType"</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_0mfd2ul">
+          <text>"decision_making_work"</text>
+        </outputEntry>
+      </rule>
+      <rule id="DecisionRule_03n8x3v">
+        <inputEntry id="UnaryTests_0u2qxu3">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1ps3x4j">
+          <text>"addListingDate","createHearingBundle","reviewHearingBundle","generateDraftDecisionAndReasons","uploadDecision","uploadHearingRecording","updateHearingRequirements","editListing",
+             "followUpOverdueHearingRequirements","reviewHearingRequirements"</text>
+        </inputEntry>
+        <outputEntry id="LiteralExpression_0c3eia6">
+          <text>"workType"</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_19poitv">
+          <text>"hearing_work"</text>
+        </outputEntry>
+      </rule>
+      <rule id="DecisionRule_1mgtmkd">
+        <inputEntry id="UnaryTests_0mpogvu">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0uoqs4p">
+          <text>"processApplication","processHearingRequirementsApplication","processHearingCentreApplication","processApplicationToExpedite","processApplicationToTransfer",
+"processApplicationForTimeExtension","processApplicationToWithdraw","processAppealDetailsApplication","processLinkedCaseApplication","processReinstatementApplication"</text>
+        </inputEntry>
+        <outputEntry id="LiteralExpression_1qi2rsb">
+          <text>"workType"</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_1spq9ei">
+          <text>"applications"</text>
+        </outputEntry>
+      </rule>
+      <rule id="DecisionRule_00yasz5">
+        <inputEntry id="UnaryTests_0zfo8ln">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_04wtfxs">
+          <text>"allocationFTPAToJudge","decideOnFTPA"</text>
+        </inputEntry>
+        <outputEntry id="LiteralExpression_1ywf3hm">
+          <text>"workType"</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_0jcwzi7">
+          <text>"upper_tribunal"</text>
```

Comment:
It appears we have missed access_requests worktype as per -> https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1507733332#WorkTypeFilters-WorkTypes